erpnext/accounts/doctype/pos_invoice/pos_invoice.py

```
@@ -868,9 +868,8 @@ def get_bundle_availability(bundle_item_code, warehouse):
 	for item in product_bundle.items:
 		item_bin_qty = get_bin_qty(item.item_code, warehouse)
 		item_pos_reserved_qty = get_pos_reserved_qty(item.item_code, warehouse)
```

Ruff (0.12.2)

Local variable `item_pos_reserved_qty` is assigned to but never used

Remove assignment to unused variable `item_pos_reserved_qty`

(F841)